crates/typst-library/src/math/attach.rs

```diff
@@ -81,19 +81,18 @@ impl Packed<AttachElem> {
     }
 }
 
-/// Grouped primes.
+/// グループ化されたプライム記号。
 ///
 /// ```example
 /// $ a'''_b = a^'''_b $
 /// ```
 ///
-/// # Syntax
-/// This function has dedicated syntax: use apostrophes instead of primes. They
-/// will automatically attach to the previous element, moving superscripts to
-/// the next level.
+/// # 構文
+/// この関数には専用の構文があり、primes関数の代わりにアポストロフィー記号を使います。
+/// それらは一段階上の上付き文字のレベルに移動され、直前の要素に自動的に付加されます。
```

Comment:
良い訳が思いつかないのですが、少し不自然な表現に感じます。
```suggestion
/// これらは自動的に前の要素に結合され、上付き文字は次のレベルに移動します。
```